Examples/Python/python/acts/examples/simulation.py

```diff
@@ -394,6 +394,16 @@ def addFatras(
     s.addAlgorithm(alg)
 
     # Output
+    addFatrasWriters(s, outputDirCsv, outputDirRoot)
+
+    return s
+
+
+def addFatrasWriters(
```

Comment:
Pedantic, but 
```suggestion
def addSimWriters(
```
?

Reply:
Sorry, that's my fault for suggesting that name. `addSimWriters` is much better.